src/main/java/frc/robot/subsystems/ShooterPivot.java

```diff
@@ -97,12 +100,40 @@ private void setPosition(Rotation2d position) {
     setpoint = position.getRadians();
   }
 
+  public Rotation2d getEstimatedAngle(double distance) {
+    double angle = Constants.ShooterPivot.a * distance + Constants.ShooterPivot.b;
+    return Rotation2d.fromDegrees(angle);
+  }
+
+  public double speakerDistance(Pose2d RobotPose) {
+    if (DriverStation.getAlliance().isPresent()) {
+      if (DriverStation.getAlliance().get() == DriverStation.Alliance.Red) {
```

Comment:
Please refactor to use ternary

Reply:
Ternaries are bad practice because other people have to read your code. It's always easier to read code that flows like English

Reply:
Embrace ternaries, they are a staple in Python.... but they also read better in Python than Java